provider/lxd/credentials.go

```diff
@@ -25,14 +29,15 @@ import (
 )
 
 const (
-	credAttrServerCert = "server-cert"
-	credAttrClientCert = "client-cert"
-	credAttrClientKey  = "client-key"
-
-	// interactiveAuthType is a credential auth-type provided as an option to
-	// "juju add-credential", which takes the user through the process of
-	// generating a certificate credential.
-	interactiveAuthType = "interactive"
+	credAttrServerCert    = "server-cert"
+	credAttrClientCert    = "client-cert"
+	credAttrClientKey     = "client-key"
+	credAttrTrustPassword = "trust-password"
+
+	// manualAuthType is a credential auth-type provided as an option, where by
+	// you've already setup a lxd remote and you have all the certs at hand to
+	// just pass directly to juju.
+	manualAuthType = "manual"
```

Comment:
I think this is better how it was:
Certificate (I have all these certs) vs Interactive (I will generate them and do the handshake with the server via trust password).

Would current credentials be broken by this change? Not that it is that important on the dev branch.

Reply:
I think this is the key to the whole PR, I'm not sure to answer your question. I'm also undecided what to do about this.